pkg/deprecated/ecdsa.go

```diff
@@ -0,0 +1,109 @@
+package deprecated
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/sha256"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/theupdateframework/go-tuf/data"
+	"github.com/theupdateframework/go-tuf/pkg/keys"
+)
+
+func init() {
+	keys.VerifierMap.Delete(data.KeyTypeECDSA_SHA2_P256)
```

Comment:
Is this necessary? IIUC `sync.Map.Store` will clobber an existing entry (which is what we want).

Reply:
You're right, I don't need this, noted the import path from your other comment!